File: src/main/java/com/group/libraryapp/domain/user/loanhistory/UserLoanHistoryRepository.java
package com.group.libraryapp.domain.user.loanhistory;

import org.springframework.data.jpa.repository.JpaRepository;

import com.group.libraryapp.domain.user.userloanhistory.UserLoanHistory;

public interface UserLoanHistoryRepository extends JpaRepository<UserLoanHistory, Long> {

  UserLoanHistory findByBookNameAndIsReturn(String bookName, boolean isReturn);

}

File: src/main/kotlin/com/group/libraryapp/domain/user/userloanhistory/UserLoanHistoryRepository.kt
package com.group.libraryapp.domain.user.userloanhistory

import org.springframework.data.jpa.repository.JpaRepository

interface UserLoanHistoryRepository : JpaRepository<UserLoanHistory, Long> {

    fun findByBookNameAndIsReturn(bookName: String, isReturn: Boolean): UserLoanHistory?

}